cmd/mt-fakemetrics/cmd/parrot.go

```diff
@@ -0,0 +1,102 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/grafana/metrictank/schema"
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"time"
+)
+
+var (
+	parrotOrgId          int
+	parrotPartitionCount int32
+	partitionMethod      schema.PartitionByMethod
+)
+
+func init() {
+	rootCmd.AddCommand(parrotCmd)
+	parrotCmd.Flags().IntVar(&parrotOrgId, "parrot-org-id", 1, "org id to publish parrot metrics to")
+	parrotCmd.Flags().Int32Var(&parrotPartitionCount, "parrot-partition-count", 128, "number of partitions to publish parrot metrics to")
+}
+
+var parrotCmd = &cobra.Command{
+	Use:   "parrot",
+	Short: "generate deterministic metrics for each metrictank partition",
+	Run: func(cmd *cobra.Command, args []string) {
+		partitionMethod = parseParrotPartitionMethod()
+		if orgs > 1 {
+			log.Fatal("parrot only works in single org mode")
+		}
+		initStats(false, "parrot")
```

Comment:
planning on reporting different stats on parrot query results, not sure if the default stats make sense or not?

Reply:
they are useful though not critical
mainly I think we should report at each point in time the number of metrics that are incorrect.